configurations/PM1735.json

```diff
@@ -0,0 +1,53 @@
+{
+    "Bus": "$bus",
+    "Exposes": [
+        {
+            "Address": "$address",
+            "Bus": "$bus",
+            "Name": "Samsung PM1735 $index FRU",
+            "Type": "EEPROM"
+        },
+        {
+            "Address": "0x6a",
+            "Bus": "$bus",
+            "Name": "Samsung PM1735 $index Temp",
```

Comment:
This appears it would not give a consistent way to know the location of the nvme and thermal is planning a split-zone fan control configuration on Everest where knowing which nvme's are located on the left versus which are on the right of the chassis will need to be consistent.

Reply:
I agree, I think if a system were booted with what is usually $index = 0 missing, it would change all the names.

Reply:
So @msbarth and @spinler what I think I am hearing is the NVMe's physical location
is what is important and not that it is say the 5th NVMe in the system

Reply:
@BMC-Bruce correct

Reply:
The physical location of a drive should be exposed by an association from the drive to its connector on Williwakas. This isn't yet implemented. The location information shouldn't be encoded in the drive name.

Reply:
As Matt mentioned we can defer this until Everest, so I'm OK with this.